package.json

```diff
@@ -74,6 +74,7 @@
     "react-native-sound": "0.10.12",
     "react-native-swipeout": "2.3.6",
     "react-native-vector-icons": "6.0.2",
+    "react-native-watch-connectivity": "saghul/react-native-watch-connectivity#podspec-module",
```

Comment:
My podspec PR was merged, you can just depend on version `0.2.0` directly instead of the fork I created back in the day.

Reply:
I will give it a try, but it's not just about the podspec. You've made some cleanup there removing unnecessary things including the demo app and last time I remember it was causing troubles

Reply:
https://github.com/saghul/react-native-watch-connectivity/commit/d0b5003e15fad4a9cbbb883ccdffed3ab68ca9dd

Reply:
IIRC that was just we could use it from git, because the plugin is in a subdirectory.

Reply:
okay, it worked